.github/workflows/update-helm-charts-index.yml

```
@@ -0,0 +1,56 @@
+name: hashicorp/vault-helm/update-helm-charts-index
```

We've decided to only keep the last part of these even though the gh-actions-importer adds the full name
```suggestion
name: update-helm-charts-index
```